src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  color: black;
```

Comment:
Color of text in this task are not black, change it an use variables for colors.